### PR TITLE
change background colour and border of icon

### PR DESCRIPTION
**Pull request by RupinderKaurr**

### CODE REVIEW

File: stylesheet.css [hunk 1]
```diff
@@ -55,7 +55,7 @@ left:3px;
  text-align: right;
 }
 #ping{
- border: 1px solid black;
+ background-color:rgb(252,251,251);
```
Comment (sahibkaur) [inline stylesheet.css]: Use hex code. I have seen you using hex code in other places - so keep it consistent everywhere.

File: stylesheet.css [hunk 2]
```diff
@@ -55,7 +55,7 @@ left:3px;
  text-align: right;
 }
 #ping{
```
Comment (sahibkaur) [inline stylesheet.css]: What does "ping" mean here?

Comment (sahibkaur) [inline stylesheet.css]: You didn't change border-radius. It was given in the issue

File: stylesheet.css (hunk 1 above)
Comment (RupinderKaurr) [reply stylesheet.css]: ok

File: stylesheet.css (hunk 2 above)
Comment (RupinderKaurr) [reply stylesheet.css]: id

Comment (RupinderKaurr) [reply stylesheet.css]: The icon has a background color of grey (search the color) and border radius.

Comment (sahibkaur) [reply stylesheet.css]: ? Border radius tusi laya ni.

Comment (sahibkaur) [reply stylesheet.css]: It is an id but why did we use the name "ping"?

Comment (RupinderKaurr) [reply stylesheet.css]: then what should the exact name

Comment (RupinderKaurr) [reply stylesheet.css]: laya hai

Comment (sahibkaur) [reply stylesheet.css]: Meinu lbhya ni, please point me to it.

Comment (sahibkaur) [reply stylesheet.css]: Something link "twitter-icon"

**Pull request closed without merging**